packages/cli-repl/src/cli-repl.ts

```diff
@@ -259,8 +260,7 @@ class CliRepl {
     const version = this.buildInfo.version;
 
     this.repl = repl.start({
-      prompt: `$ mongosh > `,
-      ignoreUndefined: true,
+      prompt: `msh > `,
```

Comment:
@mmarcon what do you think about this more simplified prompt?

I find that when i am in a more collapsed terminal view the full `mongosh` takes up a lot of space and I wouldn't mind having a few characters back.

Reply:
There as a plan to include topology information in the prompt, which would be easy to implement but we haven't gotten around to it. For example, on the old shell indicates if it's a replset/etc and what the version of the server is, like MongoDB Enterprise >. We should make a ticket to track that and save it for post-beta?

Reply:
ahhh that's a good point. We don't have that ticket yet, but that would be useful. I just felt like just having `mongosh` was giving me nothing useful.

Reply:
I made a ticket here: https://jira.mongodb.org/browse/MONGOSH-196

For the moment just `>` seems fine to me.